tests/webview/ui/test_content.py

```diff
@@ -417,7 +417,7 @@ def test_page_with_unicode_characters_in_title_loads(webview_base_url, selenium,
     # Wait 10 seconds to see if the page reloads
     sleep(10)
     # If we don't get a StaleElementReferenceException then the page didn't reload
-    assert figure.text
+    assert figure.find_element_by_css_selector('img').get_attribute('alt')
```

Comment:
There should not be any driver code in the test. The best thing to do here is to create a property method on the page object that returns True based on the locator you have here. As per the README.md:

> If you have WebDriver APIs in your test methods, You're Doing It Wrong.

The right thing to do is to create a locator on the page object and create a property method that accesses it, returns true, or whatever the best course of action is. There may be one you could already leverage on the existing page object.